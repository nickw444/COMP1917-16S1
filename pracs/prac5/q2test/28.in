28
-10 8 8 -3 -7 -8 0 10 -2 -3 3 3 -10 -4 3 -9 -8 3 10 -5 8 -6 -4 -2 -9 3 -1 6
