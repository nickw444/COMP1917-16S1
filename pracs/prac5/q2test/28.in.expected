average of array: -0.928571
